test/system/src/test/java/io/pravega/test/system/ConsumptionBasedRetentionWithMultipleReaderGroupsTest.java

```diff
@@ -391,6 +414,123 @@ public void updateRetentionPolicyForCBRTest() throws Exception {
                 new StreamImpl(SCOPE_1, STREAM_2), 0L).join().values().stream().anyMatch(off -> off == 390));
     }
 
+    @Test
+    public void multipleControllerCBRTest() throws Exception {
```

Comment:
Can you narrow the exception to what is actually thrown?

Reply:
We are using AssertExtensions.assertEventuallyEquals method to verify the results. This is an existing method which is throwing Exception, so we have to mandatorily throw the Exception class.